jekyll/index.md

```diff
@@ -6,9 +6,25 @@ description: Model Based Testing in Informal Systems
 has_children: true
 ---
 
+# Model Based Testing
+Building a model of our software gives us a couple of elegant and efficient ways to increase confidence in its correctness.
+Modeling languages (such as [TLA+](https://lamport.azurewebsites.net/tla/tla.html) or [Quint](https://github.com/informalsystems/quint)) are supported by _model checkers_, which enable us to reason about the model's properties.
+We can specify desired properties and verify that the model satisfies them, or we can generate a large number of tests directly from the model and run them against the implementation.
```

Comment:
In general, I think this is talking more about models than model-based testing. I think it's ok, since this is the part that people struggle the most to understand. This suggestion just adds a bit more focus on the testing part.

```suggestion
We can specify desired properties and verify that the model satisfies them, and then generate a large number of tests directly from the model and run them against the implementation.
```

Reply:
This is a good point. I addressed it so that the whole website now looks more general: MBT = model-based techniques